app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
+
+    if "-d" in argv:
+        index2 = 0
+
+        for _ in range(len(argv) - 1):
+            index2 += 1
+            if argv[_: _ + 1] == ["-d"]:
+                break
+
+        directories = argv[index2:index - 1]
+else:
+    filename = "file.txt"
+
+    if "-d" in argv:
+        index = 0
+
+        for _ in range(len(argv) - 1):
```

Comment:
There is no need to do this _for_-cycle 3 times in each if-else statement. If the user provides a "-f" argument, you need to get the index and the filename. If the user provides a "-d" argument, you need to get the index and the directories. 
The index can be got in a such way, for example:
`index = sys.argv.index("-f") + 1`
The filename can be got in a such way:
`filename = sys.argv[index]` and directories via `sys.argv[index:index+1]`
That's it. Try to keep it simple, without so many extra lines of the code.